Dockerfile.mustgather

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/4.16:cli
+FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
 RUN dnf install -y --nodocs --setopt=install_weak_deps=False openshift-clients \
```

Comment:
Is that intentional? Changing from `cli` base to `base-rhel9` ?

Reply:
Yeah it was intentional, Joep previously moved it from the cli image to base-rhel9 and added the dnf install line for openshift-clients [here](https://github.com/openshift/secrets-store-csi-driver-operator/commit/3192537b3d1474621d2415e2b4adbe1509e49b7c). I saw other places where this was done as well.